packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -530,3 +544,40 @@ class _TooltipOverlay extends StatelessWidget {
     );
   }
 }
+
+class _TooltipSemanticsWidget extends SingleChildRenderObjectWidget {
+  const _TooltipSemanticsWidget({
+    Key key,
+    Widget child
+  }) : super(key: key, child: child);
+
+  @override
+  RenderObject createRenderObject(BuildContext context) {
+    return _RenderTooltipSemanticsWidget();
+  }
+}
+
+class _RenderTooltipSemanticsWidget extends RenderProxyBox {
+  _RenderTooltipSemanticsWidget({RenderBox child}) : super(child) {
+    markNeedsSemanticsUpdate();
```

Comment:
Maybe I'm just unfamiliar with these APIs but why do you call this in the constructor?

Reply:
`markNeedsSemanticsUpdate` is what triggers `assembleSemanticsNode` to run, and as soon as this node is created, and I wanted to make sure that method gets called. I'm also loosely following the `_RenderScrollSemantics` example, which adds a listener to call `markNeedsSemanticsUpdate` in its constructor.

https://github.com/flutter/flutter/blob/025463f4632dbce15da7749d5d3883bea4e6457b/packages/flutter/lib/src/widgets/scrollable.dart#L726

It's possible that this is unnecessary. @goderbauer Do you think this call is safe to remove?

Reply:
This seems unnecessary. Newly created RenderObjects will always receive a semantics update when they first render (if semantics are turned on). What breaks if you remove this?